xtra-libp2p-rollover/src/v_2_0_0/maker.rs

```diff
@@ -135,11 +135,10 @@ where
                 cfd.start_rollover_maker(propose.from_commit_txid)
             })
             .await
+            .context("Rollover failed after handling taker proposal")
         {
             Ok(base_dlc_params) => base_dlc_params,
             Err(e) => {
-                tracing::warn!(%order_id, "Rollover failed after handling taker proposal: {e:#}");
```

Comment:
Same: Not sure if it would not be better to let the protocol be in charge of the log rather than the model.